src/screens/ShotsScreen.tsx

```diff
@@ -10,15 +12,24 @@ const ShotsScreen: React.FC = () => {
             <View style={styles.titleRow}>
                 <Text style={styles.title}>My Shots</Text>
             </View>
+            <FlatList
+                data={getPhotosForUser("")}
+                horizontal={false}
+                numColumns={3}
+                renderItem={({ item }) => (
+                    <PhotoGalleryThumbnail photo={item} />
+                )}
+                style={styles.imageGrid}
+            />
         </View>
+
```

Comment:
Just be careful about leaving empty lines around. A small nit though, so don't worry too much about it. It would be good if you fixed this before merging the PR :D